pytest.ini

```diff
@@ -4,6 +4,9 @@ addopts=
 	--doctest-modules
 	--doctest-glob=pkg_resources/api_tests.txt
 	-r sxX
+
+	# `pytest-xdist`:
+	-n auto
```

Comment:
I'd prefer if this could use [pytest-enabler](https://pypi.org/project/pytest-enabler), because otherwise, it becomes difficult to turn this feature off or customize the setting for a particular run. e.g.:

```
diff --git a/pyproject.toml b/pyproject.toml
index 0bc2a46f4..4e80bdc1a 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -24,6 +24,9 @@ addopts = "--flake8"
 [pytest.enabler.cov]
 addopts = "--cov"
 
+[pytest.enabler.xdist]
+addopts = "-n auto"
+
 [tool.towncrier]
     package = "setuptools"
     package_dir = "setuptools"
diff --git a/pytest.ini b/pytest.ini
index df30b8227..03fc773cf 100644
--- a/pytest.ini
+++ b/pytest.ini
@@ -4,9 +4,6 @@ addopts=
 	--doctest-modules
 	--doctest-glob=pkg_resources/api_tests.txt
 	-r sxX
-
-	# `pytest-xdist`:
-	-n auto
 doctest_optionflags=ALLOW_UNICODE ELLIPSIS
 # workaround for warning pytest-dev/pytest#6178
 junit_family=xunit2
```

However, when I applied the change, it interacts badly with pytest-cov. It works with pytest-cov disabled. I'll investigate.

Reply:
Fixed in 7fe4a4054.